nx-dev/ui-blog/src/lib/authors.tsx

```diff
@@ -21,7 +23,7 @@ export function BlogAuthors({
             src={`/documentation/blog/images/authors/${author.name}.jpeg`}
             className="relative inline-block h-6 w-6 rounded-full ring-1 ring-white grayscale dark:ring-slate-900"
           />
-          <AuthorDetail author={author} />
+          {showAuthorDetails && <AuthorDetail author={author} />}
```

Comment:
Not related to the migration right? Just a UI tweak?

Reply:
Correct only a UI tweak.